styles.css

```diff
@@ -0,0 +1,53 @@
+html{
+    font-size: 12px;
+    background-color: beige;
+}
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.6rem;
+    line-height: 1.5;
+    text-align: center;
+    color: #333;
+    margin: 0;
+}
+h1{
+    font-size: 4rem;
+    margin bottom: 0;
+}
+@media (max-width: 460px) {
+    h1 {
+        font-size: 3.5rem;
+        line-height: 1.2;
+    }
+}
+h2 {
+    font-size: 3.25rem;
+}
+a {
+    color: #599;
+}
+#main{
+    margin:10px 10px;

```

Comment:
it would be better to write "margin: 10px;" this way your code will look professional.